h/routes.py

```diff
@@ -118,6 +118,12 @@ def includeme(config):
     config.add_route('stream_atom', '/stream.atom')
     config.add_route('stream_rss', '/stream.rss')
 
+    # Organizations
+    config.add_route('organization_logo',
+                     '/organizations/{pubid}/logo',
+                     factory='h.resources.OrganizationLogoFactory',
+                     traverse='/{pubid}')
```

Comment:
So the URLs chosen are `/organizations/{pubid}/logo`.

Reply:
P.S. This is using the `OrganizationLogoFactory` that was merged here https://github.com/hypothesis/h/pull/4902